pkg/printers/internalversion/printers.go

```diff
@@ -1193,14 +1205,19 @@ func getNodeExternalIP(node *api.Node) string {
 
 // findNodeRole returns the role of a given node, or "" if none found.
 // The role is determined by looking in order for:
+// * a node-role.kubernetes.io/master label
 // * a kubernetes.io/role label
 // * a kubeadm.alpha.kubernetes.io/role label
-// If no role is found, ("", nil) is returned
 func findNodeRole(node *api.Node) string {
```

Comment:
I think you're right, but I think here we're still undoing the bad PRs that got us to this regression.